bundles/org.openhab.binding.matter/src/main/java/org/openhab/binding/matter/internal/config/EndpointConfiguration.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.matter.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link EndpointConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Dan Cunningham - Initial contribution
+ */
+@NonNullByDefault
+public class EndpointConfiguration {
+
+    public Integer endpointId = 0;
+    public Integer pollInterval = 1800;
```

Comment:
Just a remark (and not a request for change): you could have used `int` instead of `Integer`